src/app/utils/formValidationUtils.js

```diff
@@ -156,19 +143,19 @@ function validate(form, cb) {
       validate: () => true,
       errorMsg: '',
     },
-    'starting-page': {
+    startPage: {
       validate: (val) => isNumeric(val),
       errorMsg: 'Please enter the starting page',
     },
-    'ending-page': {
+    endPage: {
       validate: (val) => isNumeric(val),
       errorMsg: 'Please enter the end page',
     },
   };
 
   const error = {};
   _mapObject(form, (val, key) => {
-    const isValid = fieldsToCheck[key].validate(val);
+    const isValid = fieldsToCheck[key].validate('' + val);
```

Comment:
This should just be `val`. Some of the checks are checking to see if a value is a number. If you want to convert to a string, it should be done in the `validate` function above.